File: Examples/Mvc.VB/Models/ProcessingViewModel.vb
Imports GleamTech.Examples
Imports GleamTech.ImageUltimate.AspNet

Namespace Models

    Public Class ProcessingViewModel

        Public Property ExampleFileSelector As ExampleFileSelector

        Public Property TaskSelectList As List(Of SelectListItem)

        Public Property ImagePath As String

        Public Property TaskAction As Action(Of ImageWebTask)

        Public Property CodeString As String

    End Class

End NameSpace

File: Examples/AspNetCoreCS/Models/ProcessingViewModel.cs
using System;
using System.Collections.Generic;
using GleamTech.Examples;
using GleamTech.ImageUltimate.AspNet;
using Microsoft.AspNetCore.Mvc.Rendering;

namespace GleamTech.ImageUltimateExamples.AspNetCoreCS.Models
{
    public class ProcessingViewModel
    {
        public ExampleFileSelector ExampleFileSelector { get; set; }

        public List<SelectListItem> TaskSelectList { get; set; }

        public string ImagePath { get; set; }

        public Action<ImageWebTask> TaskAction { get; set; }

        public string CodeString { get; set; }
    }
}
